bigtable-dataflow-parent/bigtable-hbase-beam/pom.xml

```diff
@@ -205,6 +214,13 @@ limitations under the License.
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <configuration>
+                    <usedDependencies>org.apache.commons:commons-compress,org.apache.beam:beam-sdks-java-io-hbase</usedDependencies>
```

Comment:
Please add a comment why these are here:
- compress is for security fixes
- beam-sdks-java-io-hbase is for the HBaseCoderProviderRegistrar

Also please separate the beam-sdks-java-io-hbase dependency above from the security comment and add a similar comment about the HBaseCoderProviderRegistrar